store/auth.js

```diff
@@ -120,22 +99,15 @@ export const actions = {
         })
         .then(({ data }) => data && data.login)
 
-      if (res && res.token) {
-        await this.app.$apolloHelpers.onLogin(res.token)
-        commit('SET_TOKEN', res.token)
-        delete res.token
-        commit('SET_USER', res)
-        commit('SET_PENDING', false)
-        return true
-      } else {
-        commit('SET_PENDING', false)
-        throw new Error('THERE IS AN ERROR')
-      }
+      await this.app.$apolloHelpers.onLogin(res.token)
```

Comment:
This is the only method step which I'm not familiar yet. But if we get an error here, I think it's OK just to bubble up the error to the user, no?